api/kogito-events-core/src/main/java/org/kie/kogito/event/process/MultipleProcessInstanceDataEvent.java

```diff
@@ -35,8 +35,16 @@ public MultipleProcessInstanceDataEvent(URI source, Collection<ProcessInstanceDa
     }
 
     public boolean isCompressed() {
-        Object extension = getExtension(MultipleProcessInstanceDataEvent.COMPRESS_DATA);
-        return extension instanceof Boolean ? ((Boolean) extension).booleanValue() : false;
+        return isCompressed(getExtension(MultipleProcessInstanceDataEvent.COMPRESS_DATA));
+    }
+
+    public static boolean isCompressed(Object extension) {
+        if (extension instanceof Boolean) {
+            return ((Boolean) extension).booleanValue();
```

Comment:
if it is casted to `Boolean`, do you think we need the `booleanValue` call? Isn't it redundant?
```suggestion
            return (Boolean) extension;
```

Reply:
Calling booleanValue(), implicit casting is avoided.